apps/qa/src/gru/helpers.py

```diff
@@ -54,11 +49,11 @@ def map_geosupport_version(patched_version: str) -> str:
     return f"{major}{chr(int(minor)+96)}"  ## converts 1 to 'a', 2 to 'b', etc
 
 
-def get_qaqc_runs(geosupport_version: str) -> dict[str, dict]:
-    workflows: dict[str, dict] = {}
-    raw_workflow_runs: list[dict] = []
+def get_qaqc_runs(geosupport_version: str) -> dict[str, github.WorkflowRun]:
+    workflow_runs: dict[str, github.WorkflowRun] = {}
+    raw_workflow_runs: list[github.WorkflowRun] = []
     page = 0
-    while len(workflows) != 7 and (page == 0 or (len(raw_workflow_runs) > 0)):
+    while len(workflow_runs) != 7 and (page == 0 or (len(raw_workflow_runs) > 0)):
```

Comment:
Curious what the magical 7 signifies here?

Reply:
Ah. Number of distinct workflows. This function is looking for the most recent run of each, github api returns most recent first so it terminates when they're found. I will make the change though since this number is not that magical, and I have a constant which lists all 7, so I can grab the length pretty easily since that should be determining this